packages/lumx-react/src/utils/truncate.ts

```diff
@@ -0,0 +1,41 @@
+export enum EllipsisType {
+    START = 'start',
+    END = 'end',
+    MIDDLE = 'middle',
+}
+
+export interface TruncateParams {
+    max: number;
+    ellipsisType?: EllipsisType;
+    separator?: string;
+}
+
+export const ELLIPSIS_CHARS = '...';
```

Comment:
```suggestion
export const ELLIPSIS_CHARS = '…';
```
You should use the dedicated ellipsis character instead.

Reply:
By the way, I don't think this constant is necessary and I don't think this is necessary to export it.